examples/laplace_demo/laplace.ipynb

```diff
@@ -1,21 +1,21 @@
 {
```

Comment:
<p>I cannot visualize the formulas here. Are they visible?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyDP/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<img width="954" alt="Screenshot 2021-12-14 at 5 40 45 PM" src="https://user-images.githubusercontent.com/61241031/145995901-c40b25a2-c919-489e-8f7e-a00d4449b0a5.png">
<img width="1113" alt="Screenshot 2021-12-14 at 5 40 57 PM" src="https://user-images.githubusercontent.com/61241031/145995910-18d6640a-f647-4bd1-98e2-690fa3d4a21f.png">


I can visualize the formulas on both ReviewNB and Github's rendering of the notebook. Can you please elaborate ?